docs/commands.md

```diff
@@ -53,8 +53,8 @@ This input should be used sparingly and with caution, as it involves interacting
 
 #### Subdomain Testing and `subdomain-url`
 
-The `subdomain-url` parameter is utilized when testing subdomain support in your IPFS gateway. It can be set to any domain that your gateway permits.
-During testing, the suite keeps connecting to the `gateway-url` while employing HTTP techniques to simulate requests as if they were sent to the subdomain.
+The `subdomain-url` parameter is utilized when testing subdomain support in your IPFS gateway. It can be set to any domain that your gateway dedicated and safelisted for [Subdomain gateway](https://specs.ipfs.tech/http-gateways/subdomain-gateway/) feature.
+During testing, the suite sends HTTP requests to the `gateway-url` while setting HTTP `Host` header value to be based on the parent domain name from `subdomain-url` to simulate Subdomain requests.
```

Comment:
I'm still confused why we don't just send requests to the subdomain, but this is more clear about whats being done. Maybe a slight improvement:

```suggestion
The `subdomain-url` parameter is utilized when testing subdomain support in your IPFS gateway. It can be set to any domain that your gateway dedicated and safelisted for [Subdomain gateway](https://specs.ipfs.tech/http-gateways/subdomain-gateway/) feature.
During testing, the suite sends HTTP requests to the `gateway-url` and  sets the HTTP `Host` header value to the parent domain name from `subdomain-url` to simulate Subdomain requests.
```

Reply:
The reason is that the hostname is meaningless in complex deployments of  HTTP, it is usually some internal IP, and the `Host` header is what decides where request is routed. 

We test what actually matters and decides about routing, and if that works, we know simple setups will work too.